### PR TITLE
add: how to exit vim

### PR DESCRIPTION
**Pull request by cckcoder**

### CODE REVIEW

**Review by ninefyi — APPROVED** [review]
Thank you for your contribution.
It looks good to me.

File: content/programming-cookbook/vim-and-neovim/how-exit-vim.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: How to exit vim
+showMetadata: true
+editable: true
+showToc: true
+---
+
+## How to exit vim?
+
+![demo](images/how-to-exit-vim.jpg)
```
Comment (aaronamm) [inline content/programming-cookbook/vim-and-neovim/how-exit-vim.md:10]: Nice introduction.

File: content/programming-cookbook/vim-and-neovim/how-exit-vim.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Well it be like the meme above, Loll
+
+> Don't worry We'll tell you the way how to exit vim like a charm!!
+
+* Basically you should be `Normal Mode` then press colon + q `:q`
+  * wait... if you exit vim with `:q` it not be saved what you edit.
```
Comment (aaronamm) [inline content/programming-cookbook/vim-and-neovim/how-exit-vim.md]: Very small change, please use capital W.

**Review by aaronamm — CHANGES_REQUESTED** [review]
Please update your PR from my feedback. Then I will merge it.

Thank you.

**Review by aaronamm — APPROVED** [review]

**Pull request merged**